hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OmMetadataManagerImpl.java

```diff
@@ -193,6 +195,8 @@ public class OmMetadataManagerImpl implements OMMetadataManager,
    * |----------------------------------------------------------------------|
    * |  snapshotInfoTable | /volume/bucket/snapshotName -> SnapshotInfo     |
    * |----------------------------------------------------------------------|
+   * |  renamedKeyTable   |  objectID -> RepeatedString                     |
```

Comment:
We need to also make sure that during snapshot creates, we block renames.

Reply:
Locking will be done in the follow-up jira that actually uses this table in SDT and KDT